packages/react-theme/src/alias/highContrast.ts

```diff
@@ -104,24 +104,23 @@ export const neutralColorTokens: NeutralColorTokens = {
   neutralShadowKeyDarker: 'rgba(0,0,0,0.48)', // rgba(0,0,0,0.48) undefined
   brandShadowAmbient: 'rgba(0,0,0,0.30)', // rgba(0,0,0,0.30) undefined
   brandShadowKey: 'rgba(0,0,0,0.25)', // rgba(0,0,0,0.25) undefined
-};
+});
 
 export const generateSharedColorTokens: (
   globalSharedTokens: GlobalSharedColors,
-) => Record<keyof GlobalSharedColors, SharedColorTokens> = globalSharedTokens => {
-  return Object.keys(globalSharedTokens)
-    .filter(sharedColor => sharedColor !== 'brand' && sharedColor !== 'grey')
-    .reduce((acc, sharedColor) => {
-      acc[sharedColor as keyof GlobalSharedColors] = {
-        background1: `var(--global-color-white)`,
-        background2: `var(--global-color-black)`,
-        background3: `var(--global-color-white)`,
-        foreground1: `var(--global-color-black)`,
-        foreground2: `var(--global-color-white)`,
-        foreground3: `var(--global-color-white)`,
-        borderActive: `var(--global-color-hcHighlight)`,
-        border2: `var(--global-color-white)`,
-      };
-      return acc;
-    }, {} as Record<keyof GlobalSharedColors, SharedColorTokens>);
+  globalColors: Theme['global']['color'],
+) => Record<keyof GlobalSharedColors, SharedColorTokens> = (globalSharedTokens, globalColors) => {
+  return (Object.keys(globalSharedTokens) as Array<keyof GlobalSharedColors>).reduce((acc, sharedColor) => {
```

Comment:
Do you need a copy of the same object for every single colour ?

Reply:
Theoretically no, but as this is (supposed to be) generated by a token pipeline, the pipeline does not know that global shared tokens are not used and all the objects are the same.